README.md

```diff
@@ -192,6 +192,40 @@ import * as stats from './stats';
 export { fallback } from './fallback';
 ```
 
+## How to add allowed word
+
+All the allowed words are located in an array in ` src/config.ts ` inside the object ` ALEX ` as shown below. 
+
+```ts
+ALEX: {
+    profanitySureness: 2,
+    noBinary: true,
+    allow: [
+      'just',
+      'brother-sister',
+      'brothers-sisters',
+      'daft',
+      'master',
+      'clearly',
+      'moaning',
+      'host-hostess',
+      'husband-wife',
+      'obvious',
+      'simple',
+      'of-course',
+      'special',
+      'dad-mom',
+      'fellowship',
+      'basically',
+      'long-time-no-see',
+    ],
+  },
+
+```
+
+In order to add a new allowed word you have to do it inside of single quotes (` '' `) and for phrases (multiple words togheter) you should put a hiphon (` - `).
```

Comment:
This should be changed that to add a rule it must match what is expected for a rule.

Here is the rule list:
https://github.com/retextjs/retext-profanities/blob/main/rules.md
https://github.com/retextjs/retext-equality/blob/main/rules.md

Reply:
I think we could just add these links to the documentation because they are mostly lists of words that are not allowed and patterns that are not allowed.
What do you think about this?

Reply:
Yes, please add the links to the docs.